xen/xenops_types.ml

```diff
@@ -27,12 +27,14 @@ module Vgpu = struct
   (* Example for nvidia:
   {
     physical_pci_address : None
-    virtual_pci_address : Some 0000:00:0a.0
+    config_file: ""
+    virtual_pci_address : Some {domain; bus; device; fn}
     type_id: 45
   }
   *)
   type nvidia = {
     physical_pci_address: address option; (* unused; promoted to Vgpu.t *)
+    config_file: string;
```

Comment:
I think that it will work if you make this a `string option`, just like the `physical_pci_address` became an `address option`.

Reply:
Absolutely.
Thanks, updated.